main_test.go

```diff
@@ -217,4 +217,56 @@ func TestDownload_Chunks(t *testing.T) {
 	}
 }
 
-// TODO: add tests for getDownloadSize (success with Content-Length, success with Content-Range, failure)
+func TestGetDownloadSize_ContentLength(t *testing.T) {
+	s := httptest.NewServer(http.HandlerFunc(
+		func(w http.ResponseWriter, r *http.Request) {
+			fmt.Fprint(w, "Test")
+		},
+	))
+	defer s.Close()
+
+	d := DefaultDownloader()
+	d.client = &http.Client{Timeout: d.TimeoutPerChunk}
+	got, err := d.getDownloadSize(context.Background(), s.URL)
+
+	if err != nil {
+		t.Errorf("encountered an error:%s", err)
```

Comment:
NIT (if you're changing it, feel free to do it on the next test cases too)

```suggestion
		t.Errorf("expected no error getting the file size, got %s", err)
```